### PR TITLE
Bkj cron emails

### PR DESCRIPTION
**Pull request by bak45247**
## Description

Changed cron emails to utilize a template rather than a python string

Fixes # 223

## Size and Type of change

- Small Change - 1 person needs to review this
- Medium Change - 2+ people need to review this

- Bug fix

## How Has This Been Tested?

Not much testing as the template is essentially the same as the appointment signup template

## Checklist:

Only check what applies and what you have done.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have tested on multiple browsers (Chrome, Firefox, Safari, IE suite)

### CODE REVIEW

File: writing_center/cron/__init__.py [hunk 1]
```diff
@@ -47,32 +47,24 @@ def send_reminder_emails(self):
...
             upcoming_appointments = self.cron.get_upcoming_appointments()
             for appointment in upcoming_appointments:
-                student = self.cron.get_user(appointment.student_id)
                 tutor = self.cron.get_user(appointment.tutor_id)
+                student = self.cron.get_user(appointment.student_id)
+                subject = "Your Upcoming Appointment"
+
+                appt_info = {'date': appointment.scheduledStart.date().strftime("%m/%d/%Y"),
+                             'time': appointment.scheduledStart.time().strftime("%I:%M %p"),
+                             'tutor': tutor.firstName + ' ' + tutor.lastName}
```
Comment (c-schwarze) [inline writing_center/cron/__init__.py]: 👍

File: writing_center/cron/__init__.py [hunk 2]
```diff
@@ -47,32 +47,24 @@ def send_reminder_emails(self):
         try:
             upcoming_appointments = self.cron.get_upcoming_appointments()
             for appointment in upcoming_appointments:
-                student = self.cron.get_user(appointment.student_id)
                 tutor = self.cron.get_user(appointment.tutor_id)
+                student = self.cron.get_user(appointment.student_id)
+                subject = "Your Upcoming Appointment"
```
Comment (c-schwarze) [inline writing_center/cron/__init__.py]: I don't love the subject name here. I think the previous subject name was better suited - "Writing Center Multilingual Appointment Reminder" and "Writing Center Appointment Reminder"

File: writing_center/cron/__init__.py [hunk 3]
```diff
@@ -47,32 +47,24 @@ def send_reminder_emails(self):
...
-                    subject = "Writing Center Multilingual Appointment Reminder"
-                    message = "Thank you for signing up for a multilingual writing support appointment with the " \
-                              "Writing Center. Here are the details of your appointment tomorrow:\n\nTutor: " \
-                              "{0} {1}\nStart Time: {2}\nEnd Time: {3}\nLocation: Writing Center (HC 324)\n\nIf " \
-                              "there's an error or you need to cancel, visit the Writing Center website " \
-                              "(https://tutorlabs.bethel.edu/writing-center), click View Your Appointments, and " \
-                              "click on the appointment to cancel.".format(tutor.firstName, tutor.lastName,
-                                                                           appointment.scheduledStart,
-                                                                           appointment.scheduledEnd)
+                    self.mail.send_message(subject, render_template('emails/upcoming_email_multilingual.html', **locals()), student.email, cc='', bcc='', html=True)
```
Comment (c-schwarze) [inline writing_center/cron/__init__.py]: can we leverage the same template file and just update lines 54-56 to include the differing variables?

Comment (bak45247) [reply writing_center/cron/__init__.py]: yeah, i can make both of these changes real quick

File: writing_center/cron/__init__.py (hunk 2 above)
Comment (bak45247) [reply writing_center/cron/__init__.py]: done

File: writing_center/cron/__init__.py (hunk 3 above)
Comment (bak45247) [reply writing_center/cron/__init__.py]: done

**Pull request merged**